samples/python/camera_calibration_show_extrinsics.py

```diff
@@ -188,7 +188,7 @@ def main():
 
     fig = plt.figure()
     ax = fig.gca(projection='3d')
-    ax.set_aspect("equal")
+    ax.set_aspect("auto")
```

Comment:
@APrigarina 

What is the reason of this change?

Before:

![image](https://user-images.githubusercontent.com/8229425/123923701-84ff4180-d989-11eb-8648-c4dd6bd62086.png)

After:

![image](https://user-images.githubusercontent.com/8229425/123923781-9b0d0200-d989-11eb-9b52-696db3b6a4aa.png)

Reply:
@catree, this is because of matplotlib bug in versions >=3.1.0 on Windows. The picture depends on window size, the graph is still valid. On latest versions of matlotlib the picture doesn't change with window resizing
![image](https://user-images.githubusercontent.com/44146733/123925672-d14f7f00-d993-11eb-8b22-cc75dd477660.png)

Reply:
@APrigarina 

Ok, I will trust your change. Any possibility to make it looks good on any platform regardless of the Matplotlib version would be welcome.